src/styles/blocks/moyo.scss

```
@@ -0,0 +1,81 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+.header {
+  line-height: 14px;
+  grid-area: header;
+  padding: 0 50px;
+  background-color: white;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  align-items: center;
+
+  &__container {
+    justify-content: space-between;
+    display: flex;
+  }
+}
+
+.logo {
+  margin: 10px auto;
+  width: 40px;
+
+  &_link {
```

```suggestion
  &__link {
```
if it's an element you should use 2 underscores